app/controllers/agent/ManageAgentController.scala

```diff
@@ -140,13 +140,13 @@ class ManageAgentController @Inject()(
         }
       }, { success =>
         success.manageAgentOption match {
-          case AssignToSomeProperties => Future.successful(joinOldJourney(agentCode))
+          case AssignToSomeProperties => Future.successful(joinOldAgentAppointJourney(agentCode))
           case AssignToAllProperties =>
             Future.successful(Ok(addAgentToAllProperties(submitAgentAppointmentRequest, success.agentName, agentCode)))
           case UnassignFromAllProperties =>
             Future.successful(
               Ok(unassignAgentFromAllProperties(submitAgentAppointmentRequest, success.agentName, agentCode)))
-          case _ => ??? //fixme this will be implemented in subsequent stories (VTCCA-3206)
+          case UnassignFromSomeProperties => Future.successful(joinOldRevokeAppointJourney(agentCode))
```

Comment:
join?

Reply:
Karen said we would just join the old journey for the "some" options

Reply:
is this exhaustively matched? We no longer need a catch-all clause `case _ => ` right? That's good, I am just checking...